cmd/broker/provisioning.go

```diff
@@ -84,6 +84,13 @@ func NewProvisioningProcessingQueue(ctx context.Context, provisionManager *proce
 			stage:     createRuntimeStageName,
 			step:      provisioning.NewGenerateRuntimeIDStep(db.Operations(), db.RuntimeStates(), db.Instances()),
 		},
+		// postcondition: operation.RuntimeID is set
+		{
+			condition: provisioning.SkipForOwnClusterPlan,
+			stage:     createRuntimeStageName,
+			step:      provisioning.NewCreateKymaNameStep(db.Operations()),
```

Comment:
Is this steps going to do anything else? For now it feels like it is just doing one assignment? I do not think that it is necessary to create a new step for such assignment. Maybe we could put that in initialisation step or somewhere else?

Reply:
This is how we decided to approach after discussion.

Reply:
See https://github.com/kyma-project/kyma-environment-broker/issues/905#issuecomment-2230587326

Reply:
In my opinion this approach is justified, if you have doubts we can arrange a meeting tomorrow.
Some pros:
- simple logic, 
- postcondition guaranteed,
- easy to manage in the process of migration from Provisioner to KIM - we will remove some steps gradually
- easy to extend
- easy to test

Currently we have a lot of quite complex steps, and it is hard to grasp what is set where, could we remove step, could we add a new one, and where to put it, to have all required data i to provide all required data for subsequent steps.

Reply:
the question is where we should put this assignment. What are hard requirements? Such assignment (logic, which calculates the Kyma resoruce name) must:
1. be executed after runtime ID gathering/generation (after interaction with Provisioner)
2. be before Runtime CR creation which must be before ApplyKyma step (currently the logic is in the apply kyma step)

@ralikio  - I know this logic is very simple, but we can't put it into initialisation step or something like this (it is too early).
We can't put it into a provisioner interaction step (we gather runtime ID there), because this step will be removed soon.